crates/fluvio-version-manager/src/command/install.rs

```diff
@@ -0,0 +1,285 @@
+//! Install Command
+//!
+//! Downloads and stores the sepecific Fluvio Version binaries in the local
+//! FVM cache.
+
+use std::fs::{File, create_dir, copy, write};
+use std::io::{Cursor, Read};
+use std::path::PathBuf;
+
+use anyhow::{Error, Result};
+use clap::Parser;
+use colored::Colorize;
+use serde::{Deserialize, Serialize};
+use semver::Version;
+use sha2::{Digest, Sha256};
+use surf::StatusCode;
+use tempfile::TempDir;
+use url::Url;
+
+use fluvio_hub_util::HUB_REMOTE;
+use fluvio_hub_util::fvm::{Client, PackageSet, DEFAULT_PKGSET, Channel};
+
+use crate::GlobalOptions;
+use crate::common::TARGET;
+use crate::common::fluvio::{fluvio_bin_path, create_fluvio_dir};
+use crate::common::notify::Notify;
+use crate::common::workdir::fvm_versions_path;
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct VersionManifest {
+    pub channel: Channel,
+    pub version: Version,
+}
+
+/// The name printed to CLI in place of `default` for the Package Set
+const DEFAULT_PACKAGE_SET_DISPLAY_NAME: &str = "fluvio";
+
+/// The name of the manifest file for the Package Set
+const PACKAGE_SET_MANIFEST_FILENAME: &str = "manifest.json";
+
+/// The `install` command is responsible of installing the desired Package Set
+#[derive(Debug, Parser)]
+pub struct InstallOpt {
+    #[command(flatten)]
+    global_opts: GlobalOptions,
+    /// Version to install
+    #[arg(index = 1, default_value_t = Channel::Stable)]
+    version: Channel,
+    /// Registry used to fetch Fluvio Versions
+    #[arg(long, env = "HUB_REGISTRY_URL", default_value = HUB_REMOTE)]
```

Comment:
Since another command will use this, move into common arg.   Now show stopper for this PR though

Reply:
Good catch! Will move it to a global scope!

Reply:
Here I added to the global scope: 0e39b0c0bf835696ad99b8e1f4975e68dd7dda95